go-ipfs/Dockerfile

```diff
@@ -97,4 +98,3 @@ ENV IPFS_LOGGING ""
 ENTRYPOINT ["/sbin/tini", "--", "/usr/local/bin/start_ipfs"]
 
 CMD ["daemon", "--migrate=true", "--agent-version-suffix=docker"]
-
```

Comment:
nit: I don't think this line should be removed (files should be end with new line).

Reply:
There were two LFs at the end of the file, so I removed one.
```
$ git checkout main
Already on 'main'
Your branch is up to date with 'origin/main'.
$ hexdump -C go-ipfs/Dockerfile | tail -n 3
000009e0  6e 2d 73 75 66 66 69 78  3d 64 6f 63 6b 65 72 22  |n-suffix=docker"|
000009f0  5d 0a 0a                                          |]..|
000009f3
```